src/psyclone/psyGen.py

```diff
@@ -568,7 +568,7 @@ def __init__(self, alg_invocation, idx, schedule_class,
             If not None, this number is added to the name ("invoke_").
         :type idx: Integer.
         :param schedule_class: The schedule class to create for this invoke.
-        :type schedule_class: InvokeSchedule class.
```

Comment:
Almost: you've missed the enclosing back ticks around the "psyclone.psyGen.InvokeSchedule"

Reply:
fixed